pyproject.toml

```diff
@@ -59,6 +59,7 @@ deltalake = { version = ">=0.6.4,<0.10.0", optional = true }
 
 # Distributed
 modin = { version = "^0.20.1", optional = true }
+s3fs = { version = "0.4.2", optional = true }
```

Comment:
Why would we need s3fs as a required dependency when installing Modin? Not even Modin itself requires it

Reply:
It's a fair default for Modin to keep it optional because you might not be using s3 at all. I added this here because modin load tests are failing with missing s3fs

Reply:
But we only need it for our load tests, none of our actual awswrangler methods actually use s3fs. In the load test, it's only modin specific tests (i.e. no awswrangler involved that use it). So I don't think we want to add a dependency on users if we don't need it

Reply:
Ok, agreed. I will put it into the infra instead